mineral-ui/mineral-ui
# 356 tooltip

### Description
there are 3 commits here with multiple proposed changes

1. add flow checking to webpack dev server so that compile and type errors both go in the terminal
2. while developing Tooltip, I needed to be able to customize props and theme vars for subcomponents of Popover
3. added a tooltip component

### Motivation and context


connects #356 - closed on publish

### Screenshots, videos, or demo, if appropriate


https://356-tooltip--mineral-ui.netlify.com/

### How to test


1. open netlify preview in supported browsers
2. try using different children as the tooltip trigger
3. try putting different contents in the tooltips
4. make sure the delayed open feels right
5. use keyboard to open and close tooltips, use escape and tab off
5. play with screenreaders and make sure announcing is correct
6. read the copy and make sure it all makes sense and communicates well

### Types of changes

- New feature (non-breaking change which adds functionality)
- Other (provide details below)
chore for webpack flow

### Checklist




### How does this PR make you feel?

![harder than anticipated](https://media.giphy.com/media/w2hcGq8bbjcBy/giphy.gif)
